libraries/tools/kotlin-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/compilerRunner/GradleCompilerRunnerWithWorkers.kt

```diff
@@ -45,7 +44,7 @@ internal class GradleCompilerRunnerWithWorkers(
         workQueue.submit(GradleKotlinCompilerWorkAction::class.java) { params ->
             params.compilerWorkArguments.set(workArgs)
             if (taskOutputsBackup != null) {
-                params.taskOutputs.from(taskOutputsBackup.outputs)
+                params.taskOutputs.set(taskOutputsBackup.outputs)
```

Comment:
@hungvietnguyen why you've added changes in this file? :thinking:

Reply:
@Tapchicoma It's a small cleanup for the previous PR: I wrote it in [pull/4648](https://github.com/JetBrains/kotlin/pull/4648/files#diff-d9843c49302b36648fa62dde701c0810fda3e1f8231bcc6bf2eca14325e1c296), but when that PR was merged, that change was lost (not sure if it was intentional or a mistake by the merger), so I re-include it in this PR. Is it causing some issues?

Reply:
I am curious - what is the point of changes in this file?

Reply:
The goal is to convert `val taskOutputs: ConfigurableFileCollection` in `GradleKotlinCompilerWorkParameters` to `val taskOutputs: ListProperty<File>`, to simplify the type to only its essence (a list of file paths). It's also a convention for parameters of Gradle work actions to be of type `Property<T>`.

Reply:
> It's also a convention for parameters of Gradle work actions to be of type Property<T>.

I haven't found anything related either in [Workers API overview](https://docs.gradle.org/current/userguide/worker_api.html) documentation or in [WorkParameters](https://docs.gradle.org/current/javadoc/org/gradle/workers/WorkParameters.html) documentation. If you check `WorkParameters` documentation - it has an example with `ConfigurableFileCollection` so it is fine to use.

Would be nice in the future to include into changeset only changes related to the topic.

Reply:
I see. I thought a `ListProperty<File>` would be nicer, but it's a minor cleanup, so please revert if you think it'll keep the PR more focused.

> Would be nice in the future to include into changeset only changes related to the topic.

I 100% agree. 

But just to give a bit context on why I did this:
  - This change was originally part of https://github.com/JetBrains/kotlin/pull/4648/. And it *was* relevant there, because I was touching some related code.
  - Probably due to conflict resolution, that particular change was lost during merging (along with some other stuff). So I included the remnants of the previous PR in this PR. Originally I had a separate commit called "clean-up", and squashed them only after Ivan had reviewed it. (Previous PRs were normally squashed by the committer, so I thought squashing them in advance would make it easier for the committer.)
  - Ideally, I should have a separate PR for cleanup, but the overhead of merging a PR is a bit high, so I decided to include the cleanup in the same PR.
  
Next time I probably will just leave the cleanup in a separate commit, so it's easier to review, and let the committer do the squashing then.

I see you just merged the PR. Thanks a lot for your help, Yahor! :)